src/lib/chips/chip-list.ts

```diff
@@ -432,9 +432,9 @@ export class MdChipList implements MdFormFieldControl<any>, ControlValueAccessor
 
     let isPrevKey = (code === (isRtl ? RIGHT_ARROW : LEFT_ARROW));
     let isNextKey = (code === (isRtl ? LEFT_ARROW : RIGHT_ARROW));
-    let isBackKey = (code === BACKSPACE || code == DELETE || code == UP_ARROW || isPrevKey);
-    // If they are on an empty input and hit backspace/delete/left arrow, focus the last chip
-    if (isInputEmpty && isBackKey) {
+    let isDeleteKey = code === BACKSPACE || code == DELETE;
+    // If they are on an empty input and hit backspace/delete, focus the last chip
```

Comment:
I'm not sure the delete key should do this in addition to backspace. In both Drive and GMail, the delete key just does nothing when your focus is on the empty input.

Reply:
Removed delete key. Thanks for review!